src/main/java/com/direwolf20/buildinggadgets/client/gui/radialmenu/ModeRadialMenu.java

```diff
@@ -41,45 +32,38 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * This class was adapted from code written by Vazkii for the PSI mod: https://github.com/Vazkii/Psi
+ * Psi is Open Source and distributed under the
+ * Psi License: http://psi.vazkii.us/license.php
+ */
 public class ModeRadialMenu extends GuiScreen {
 
-    private static final ResourceLocation[] signsBuilding = new ResourceLocation[]{
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/build_to_me.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/vertical_column.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/horizontal_column.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/vertical_wall.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/horizontal_wall.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/stairs.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/grid.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/surface.png")
-    };
-    private static final ResourceLocation[] signsExchanger = new ResourceLocation[]{
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/surface.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/vertical_column.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/horizontal_column.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/grid.png")
-    };
-    private static final ResourceLocation[] signsCopyPaste = new ResourceLocation[]{
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/copy.png"),
-            new ResourceLocation(BuildingGadgets.MODID,"textures/ui/paste.png")
-    };
+    //TODO move to a enum
+    private static final ImmutableList<ResourceLocation> signsCopyPaste = ImmutableList.of(
+            new ResourceLocation(BuildingGadgets.MODID, "textures/ui/copy.png"),
+            new ResourceLocation(BuildingGadgets.MODID, "textures/ui/paste.png")
+    );
+
     private int timeIn = 0;
     private int slotSelected = -1;
     private int segments;
 
     public ModeRadialMenu(ItemStack stack) {
         mc = Minecraft.getMinecraft();
-        if (stack.getItem() instanceof GadgetGeneric)
+        if (stack.getItem() instanceof GadgetGeneric) {
             setSocketable(stack);
+        }
```

Comment:
I'm guessing that the addition of brackets here is the result of formatting. If so, the formatter should be set to forgo them for one-line blocks like this.

Reply:
It is set to don't force braces, however the autocomplete still does this.
Will remove these in the next commit

Reply:
This has been marked as resolved, but they have not been removed (they have in some other classes, but not here).

Reply:
I will fix all of them in this class, however, this is not a format cleanup PR, and these braces came from someone before me. I'll never touched anything aside the text rendering lines.

Reply:
If they were not added by you then you didn't merge properly, since they don't exist on the master branch. All I'm asking is that this PR not make changes that solely constitute adding braces that don't exist on the master branch.

In this case, you've reverted what I specifically changed (namely,  the removal of the braces): 4661e2fa9b4626009ffd5cba3e5a640d89017478

Reply:
`GadgetUtils` is the last class with needlessly added brackets.